src/CookieActionComponent.cpp

```diff
@@ -112,8 +108,8 @@ void CookieActionComponent::checkAndPerformAction(double deltaTime, double total
         if (totalTime >= previousCookieTime + 0.5) {
 
             CookiePhysicsComponent *cookiePhysicsComp = new CookiePhysicsComponent();
-            BunnyRenderComponent *renderComp = new BunnyRenderComponent(cookieShape,
-               "Phong", obsidian);
+            BunnyRenderComponent *renderComp = new BunnyRenderComponent(
+               shapeManager.getShape("Sphere"), "Phong", materialManager.getMaterial("Obsidian"));
```

Comment:
Initially I thought it would be best to change up the RenderComponent to take a `std::string` instead of a `std::shared_ptr<Shape>` and `std::shared_ptr<Material>` but it seemed like a bigger change than I originally thought due to the render component actually providing an interface to get the shape. I think this is fine though.